pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -179,11 +179,7 @@ python.pkgs.pythonPackages.buildPythonApplication rec {
   '';
 
   passthru = {
-    inherit python;
```

Comment:
This thunk should not be part of the backport because it breaks backwards compat.

Reply:
(`passthru`s are part of the public interface)

Reply:
Are you talking about the `passthru.pythonPath` removal?

Reply:
Yes.

Reply:
Updated. PTAL.